### ISSUE TITLE
add option to support keep empty line at the end of file

### ISSUE THREAD

**Issue by HerrCai0907** [mike-lischke/antlr-format#9]

**Comment by jimidle** [comment]
I don't think that this is the correct description of the issue @mike-lischke - on *nix systems, the formatter will often produce an output file without the trailing newline required for valid text files. I think that the issue is that regardless of the OS, it should always ensure that the file ends with a newline. I came here to report that as a bug.

**Comment by mike-lischke** [comment]
Makes sense.

**Comment by HerrCai0907** [comment]
@jimidle So it is a os-related bug instead of a feature request? Because I don't use windows, I even don't aware it.

**Comment by jimidle** [comment]
> @jimidle So it is a os-related bug instead of a feature request? Because I don't use windows, I even don't aware it.

Yes. It is common thing with many utilities. *nix requires a newline at the end of a text file.
